keyboards/handwired/scottokeebs/scottomacrodeck/info.json

```
@@ -0,0 +1,52 @@
+{
+    "manufacturer": "ScottoKeebs",
+    "keyboard_name": "ScottoMacrodeck",
+    "maintainer": "joe-scotto",
+    "development_board": "promicro",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true
+    },
+    "build" : {
+        "lto": true,
+    },
+    "matrix_pins": {
+        "direct": [
+            ["D3", "D2", "D1", null],
+            ["D0", "D4", "C6", "D7"],
+            ["E6", "B4", "B5", "F4"]
+        ]
+    },
+    "url": "",
+    "usb": {
+        "device_version": "1.0.0",
+        "pid": "0x0001",
+        "vid": "0x534B"
+    },
+    "layouts": {
+        "LAYOUT_ortho_3x4": {
```

This isn't quite ortho 3x4.  It's missing a key for that. 
```suggestion
        "LAYOUT": {
```

---

Still outstanding.